python_envs/install_python

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+set -vx
+prefix=${prefix:-$HOME/python_envs}
+name=${name:-easy}
+yaml_file=${yaml_file:-environment.yaml}
+
+
+conda deactivate
+mamba deactivate
+deactivate
+
+set -e
+
+module purge
+module load git
+
+# nvidia-smi || ( >&2 echo  "need a gpu node to get the correct pytorch version" ; exit 1 )
+
+git_version=$(git rev-parse --verify HEAD)
+logname="hackathon_env_${git_version}_$(date +%F_%T)_install.log"
+
+if [ ! -d  ${prefix}/miniconda3 ] ; then
+mkdir -p ${prefix}/miniconda3
+wget https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Linux-x86_64.sh -O ${prefix}/miniconda3/miniconda.sh
+bash ${prefix}/miniconda3/miniconda.sh -b -u -p ${prefix}/miniconda3
+rm -f ${prefix}/miniconda3/miniconda.sh
+fi
+
+${prefix}/miniconda3/bin/conda update -n base -c conda-forge conda mamba
+
+command="${prefix}/miniconda3/bin/mamba env create --file ${yaml_file} --prefix ${prefix}/${name}"
+echo $command > ${logname}
+echo >> ${logname}
+
+$command >>${logname} 2>&1
+cp  install_kernel ${prefix}/${name}/bin/
+mv ${prefix}/${name}/bin/cdo ${prefix}/${name}/bin/cdo.do_not_use
```

Comment:
Shouldn't this be up to the user? If they include `cdo` in their environment they likely want to use it? 🤔

Reply:
Yes, and no...
Again from Levante, and cdo is included in the conda if you install the cdo py bindings (which some of our users seem to use). But that does not mean you don't want to use the system cdo...

The classic issue that `module load python` on levante will replace your cdo without you asking for it.
Removed.

Reply:
Yes, I have already discussed this with DKRZ user support (...). I would still argue that this is a Levante bug and nothing that should go into a documentation, but I am willing to be pragmatic 😉